src/TenantSecurityRequest.php

```diff
@@ -124,4 +125,27 @@ public function unwrapKey(Bytes $edek, RequestMetadata $metadata): UnwrapKeyResp
         }
         return $unwrapResponse;
     }
+
+    /**
+     * Requests the TSP to re-key an EDEK.
+     *
+     * @param Bytes $edek The encrypted document key to re-key
```

Comment:
document key should be document encryption key here as well.

Reply:
This is the wording we use across all the TSCs, so I'm going to leave this unless we decide to change them all